3cn-ecn/nantralPlatform
# React app set event calendar

# Description

Set the events in the calendar

# Tests

1. Steps to reproduce : 

- Create some events (in the first week of june 2023)
- Go to the event pages
- See the calendar

2. Expected results :

- The event appears
- It's beautiful

# Screenshots

![image](https://user-images.githubusercontent.com/97662746/215150735-332b369f-64a1-434e-9542-a34d0e50795d.png)
